autogpt_platform/autogpt_libs/autogpt_libs/supabase_integration_credentials_store/store.py

```diff
@@ -79,6 +79,13 @@
     title="Use Credits for Jina",
     expires_at=None,
 )
+unreal_credentials = APIKeyCredentials(#
+    id="66f20754-1b81-48e4-91d0-f4f0dd82145f",
+    provider="unreal",
+    api_key=SecretStr(settings.secrets.unreal_speech_api_key),
+    title="Use Credits for Unreal",
+    expires_at=None,
+)
```

Comment:
this needs to be added to the list of hidden credentials in the frontend profile

Reply:
https://github.com/Significant-Gravitas/AutoGPT/pull/8642